ddtrace/contrib/asyncio/compat.py

```diff
@@ -1,10 +1,11 @@
-import sys
+import asyncio
 
-# asyncio.Task.current_task method is deprecated and will be removed in Python
-# 3.9. Instead use asyncio.current_task
-if sys.version_info >= (3, 7, 0):
-    from asyncio import current_task as asyncio_current_task
-else:
-    import asyncio
 
-    asyncio_current_task = asyncio.Task.current_task
+def asyncio_current_task():
+    if hasattr(asyncio, "current_task"):
+        try:
+            return asyncio.current_task()
+        except RuntimeError:
+            return None
+    else:
+        return asyncio.Task.current_task()
```

Comment:
Do this once and for all like it was before it a little more efficient that doing `if` on each call.

Reply:
Ah yes, of course, good catch